src/kerberos/mod.rs

```diff
@@ -890,7 +1076,7 @@ impl<'a> Kerberos {
                 info!(encoded_ap_req_authenticator = ?encoded_auth);
 
                 // FIXME: properly negotiate mech id - Windows always does KRB5 U2U
-                let mech_id = oids::krb5_user_to_user();
+                let mech_id = oids::ms_krb5();
```

Comment:
can you keep the KRB5 U2U for connections that aren't WinRM? we know RDP always uses KRB5 U2U